load_tests/logger/log_generator.c

```diff
@@ -54,5 +55,9 @@ int main()  {
         msleep(1000+startSeconds-endSeconds);
     }
 
+    // Uncomment below two lines to Keep the generator running for 1 hour
+    // fflush(stdout);
+    // sleep(3600);
```

Comment:
can we make the running time as a static field or flag to define instead of comment and uncomment in code?

Reply:
sure